README.org

```diff
@@ -152,10 +152,8 @@
    - ~lsp-lens-show~ - Show lenses in the current file
    - ~lsp-lens-hide~ - Hide lenses in the current file
    - ~lsp-lens-mode~ (experimental) - Turn on/off lenses in the current file.
-   - ~lsp-save-logs~ - Save a trace of all messages to and from the language server to a given file. Only works when ~lsp-save-logs~ is non-nil.
```

Comment:
please update throubleshooting section, add documentation for the new variables, modes, new shortcuts, etc. Update also the bug template.